wireless_modules/back_module/main.py

```diff
@@ -13,34 +15,52 @@
 MODULE_NUM = "3"
 
 # RZERO (for calibration of MQ135) found when the sensor was first 'activated'
-RZERO = 8.62
+RZERO = 6.1
 
 # Circumference of a 700x28 bike wheel in meters.
 WHEEL_CIRCUMFERENCE = 2.136
 
-# Define Voltage divider factor for this module or leave as None to use default voltage factor
-VOLTAGE_FACTOR = None
+# Define a function to map measured voltages to "true" voltages.
+# See util/battery_lin_reg.py
+def battery_calibration(voltage):
+    return 0.710 * voltage + 0.927
```

Comment:
Honestly I think it's okay here. Think of the entire function as a single constant value which I've extracted rather than the alternative of burying it in the battery code. I think these values are clearly a linear and constant term of some relation but naming variables like `BATTERY_CALIBRATION_LINEAR_TERM` wouldn't make things easier to read. I'll expand the documentation a bit to explain how they were generated more explicitly and state that it's just a linear relation.

Reply:
Yep I agree, don't need to add heavy variable names here, but my concern with hardcoded values is how should one understand the meaning and origin of those values. I guess it is obvious that they're constant of a simple linear equation, but it might be nice to document how they were found, perhaps mention the `battery_lin_reg.py ` script.